.github/workflows/release-branch.yml

```
@@ -0,0 +1,140 @@
+            --head "release-prep-v${VERSION}" \
+            --json mergeCommit --jq '.[0].mergeCommit.oid')
+
+          if [ -z "$MERGE_SHA" ] || [ "$MERGE_SHA" = "null" ]; then
+            echo "::error::No merged prep PR found for release-prep-v${VERSION}."
+            exit 1
+          fi
+
+          echo "MERGE_SHA=${MERGE_SHA}" >> $GITHUB_ENV
+
+      - name: Create release branch with sample URL updates
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:3:1: PATCH appears unused. Verify use (or export if used externally)

(shellcheck)